docs/UserGuide.md

```diff
@@ -150,33 +154,15 @@ Examples:
 
 Deletes the specified person from Cher.
 
-Format: `delete INDEX`
-
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the Cher.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
-
-Format: `delete PHONE_NUMBER
-
-* Deletes the person with the specified `PHONE_NUMBER`
-* The phone number **must be present in the list of contacts**
-
-Examples:
-* `list` followed by `delete 98765432` deletes the person with the phone number 98765432 in the address book.
-
-Format: `delete PARTIAL_NAME`
+Format: `delete [INDEX] [n/KEYWORDS] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
 
-* Deletes the person with name matching the specified `PARTIAL_NAME`
-* The partial name **must match at least oner person's name**
-* If multiple persons matching the partial name are found, a list of the matching persons is displayed
+* Deletes the person with the specified attribute
```

Comment:
Maybe we should mention the attribute have to be full name? since now the delete command only delete by the first name but not the last name, same applies to all the other attributes